eZ Platform/APIs/GraphQL/GraphQL_REST.md

```diff
@@ -0,0 +1,44 @@
+# GraphQL interface in eZ Platform
+
+As a developer I would like to query the content repository using the GraphQL language.
```

Comment:
This is exploration into requirements @mere-teresa, that bundle might be basis for implementation.